core-java-modules/core-java-14/src/test/java/com/baeldung/java14/foreign/api/ForeignMemoryUnitTest.java

```diff
@@ -0,0 +1,74 @@
+package com.baeldung.java14.foreign.api;
+
+import jdk.incubator.foreign.*;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.BlockJUnit4ClassRunner;
+
+import java.lang.invoke.VarHandle;
+import java.nio.ByteOrder;
+
+import static org.hamcrest.core.Is.*;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.*;
+import static org.junit.Assert.assertThat;
+
+public class ForeignMemoryUnitTest {
+
+    @Test
+    public void whenValueIsSetThenAccessTheValue(){
```

Comment:
Missing ` ` between `{}` and `{`

Reply:
Added space between `()` and `{` in all unit test methods.